src/main.ts

```diff
@@ -22,8 +27,9 @@ export class Minifier {
   private lastGeneratedPropName: string = '';
   private typeChecker: ts.TypeChecker;
   private errors: string[] = [];
+  private minifierOptions: MinifierOptions = {};
 
-  constructor() {}
+  constructor(options: MinifierOptions = {}) { this.minifierOptions = options; }
```

Comment:
you can declare this as:

```ts
constructor(private options: MinifierOptions = {}) {}
```

Then you need neither the field declaration nor the assignment within the ctor.